enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -1244,72 +1223,66 @@ func cowChunkSizeBytes() int64 {
 	return int64(os.Getpagesize() * cowChunkSizeInPages)
 }
 
-// hotSwapWorkspace unmounts the workspace drive from a running firecracker
-// container, updates the workspace block device to an ext4 image pointed to
-// by chrootRelativeImagePath, and re-mounts the drive.
-func (c *FirecrackerContainer) hotSwapWorkspace(ctx context.Context, execClient vmxpb.ExecClient) error {
+// updateWorkspaceDriveToEmptyFile updates the workspace device to point to an
+// empty file, which is located at /empty.bin relative to the chroot dir.
+//
+// This lets us avoid saving the workspace contents as part of the VM snapshot,
+// which would be wasteful because the workspace contents aren't needed for the
+// next action to be executed in the VM.
```

Comment:
nit add: "because we create a clean workspace for each action"

Reply:
Done